src/routers/UpdateUserRouter.ts

```diff
@@ -13,8 +14,7 @@ class UpdateUserRouter extends ApplicationRouter<void> {
 
   async content(req: Request): Promise<void> {
     const { firstName, lastName, netID } = req.body;
-    const currentNetID = req.query.netID;
-    await UserRepo.updateUser(currentNetID, firstName, lastName, netID);
+    await UserRepo.updateUser(req.user, firstName, lastName, netID);
```

Comment:
Same thing here with requiring the entire user in the post body when it may not be necessary.

Reply:
The user should be found from the authorization token, this happens with the authentication middleware in Authenticate.ts

Reply:
Gotcha, makes sense.